MidtransCoreKit/MidtransCoreKit/MidtransConstant.h

```diff
@@ -129,10 +130,10 @@ static NSString *const MIDTRANS_UAT_BIN_URL = @"https://app.stg.midtrans.com/sna
 
 /*staging*/
 
-static NSString *const MIDTRANS_STAGING_API_URL = @"https://api.stg.midtrans.com/v2";
+static NSString *const MIDTRANS_STAGING_API_URL = @"https://midtrans-checkout.stg.veritrans.co.id/v2";
 static NSString *const MIDTRANS_STAGING_MIXPANEL = @"cc005b296ca4ce612fe3939177c668bb";
-static NSString *const MIDTRANS_STAGING_SNAP = @"https://app.stg.midtrans.com/snap/v1";
-static NSString *const MIDTRANS_STAGING_BIN_URL = @"https://app.stg.midtrans.com/snap/v1/bank_bins";
+static NSString *const MIDTRANS_STAGING_SNAP = @"https://midtrans-checkout.stg.veritrans.co.id/snap/v1";
+static NSString *const MIDTRANS_STAGING_BIN_URL = @"https://midtrans-checkout.stg.veritrans.co.id/snap/v1/bank_bins";
```

Comment:
this URL for temporary or permanent we will change to this ?

Reply:
this is temporary, since the current staging url is using this one.

Reply:
noted